abstract/myst.yml

```
@@ -44,10 +51,10 @@ project:
 site:
   template: article-theme
   actions:
-    # - title: pdf
-    #   url: ARTICLE-NAME.pdf
-    #   static: true
-    # - title: slides
-    #   url: presentation/SLIDES-NAME.pdf
-    #   static: true
+    - title: poster
+      url: presentation/AGU2023_Jorge_eposter.pdf
+      static: true
+    - title: slides
+      url: presentation/AGU2023_Jorge.pptx
```

Let's provide links to the pdf version of the slides rather than the pptx (we can still keep the pptx in the presentation folder for archival purposes, but for the website, lets distribute the pdf) 
```suggestion
      url: presentation/AGU2023_Jorge.pdf
```